scss/_patterns_side-navigation.scss

```diff
@@ -472,6 +472,27 @@
   .p-side-navigation__item--title .p-side-navigation__link {
     color: $color-sidenav-text-active;
   }
+
+  // styles applied when side nav is used in application layout
+  @if mixin-exists(vf-application-layout--when-collapsed) {
```

Comment:
I don't understand this - when the would the mixin not exist?

Reply:
When someone imports only the side navigation, but not application layout.
Rare case probably, but may happen if some project uses very limited part of Vanilla.

Reply:
Our standalone examples are an example, where we import only base and necessary components.

Reply:
:+1: cool, just wasn't sure.